Issue in building kustomize and controller-gen with `go get: disabled by -mod=vendor`

Issue in building kustomize and controller-gen with `go get: disabled by -mod=vendor`

~~~
GOPATH=/root/golang
PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/root/bin:/usr/local/go/bin:/root/golang/bin:/usr/local/kubebuilder/bin
~~~

~~~
[root@openshift-jumpserver-0 ~]# go version
go version go1.13.15 linux/amd64
~~~

Start from a clean sheet:
~~~
rm -Rf golang
mkdir golang
~~~

Get source:
~~~
go get github.com/openshift/sriov-network-operator
cd $GOPATH/src/github.com/openshift/sriov-network-operator/
~~~

Run `make deploy-setup`:
~~~
[root@openshift-jumpserver-0 sriov-network-operator]# make deploy-setup
which: no controller-gen in (/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/root/bin:/usr/local/go/bin:/root/golang/bin:/usr/local/kubebuilder/bin)
which: no kustomize in (/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/root/bin:/usr/local/go/bin:/root/golang/bin:/usr/local/kubebuilder/bin)
go: creating new go.mod: module tmp
go get: disabled by -mod=vendor
make: *** [Makefile:127: controller-gen] Error 1
~~~

What fails here is the build of `kustomize` and `controller-gen`:
~~~
[root@openshift-jumpserver-0 sriov-network-operator]# make controller-gen
which: no controller-gen in (/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/root/bin:/usr/local/go/bin:/root/golang/bin:/usr/local/kubebuilder/bin)
which: no kustomize in (/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/root/bin:/usr/local/go/bin:/root/golang/bin:/usr/local/kubebuilder/bin)
go: creating new go.mod: module tmp
go get: disabled by -mod=vendor
make: *** [Makefile:127: controller-gen] Error 1
[root@openshift-jumpserver-0 sriov-network-operator]# make kustomize
which: no controller-gen in (/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/root/bin:/usr/local/go/bin:/root/golang/bin:/usr/local/kubebuilder/bin)
which: no kustomize in (/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/root/bin:/usr/local/go/bin:/root/golang/bin:/usr/local/kubebuilder/bin)
go: creating new go.mod: module tmp
go get: disabled by -mod=vendor
make: *** [Makefile:142: kustomize] Error 1
~~~

The problem is this here in the Makefile:
~~~
[root@openshift-jumpserver-0 sriov-network-operator]# grep GOFLAGS Makefile 
export GOFLAGS=-mod=vendor
[root@openshift-jumpserver-0 sriov-network-operator]# grep GO111 Makefile 
export GO111MODULE=on
~~~

So, I had to modify the Makefile:
~~~
[root@openshift-jumpserver-0 sriov-network-operator]# diff Makefile.old Makefile
132c132
< 	go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.3.0 ;\
---
> 	GOFLAGS="" go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.3.0 ;\
147c147
< 	go get sigs.k8s.io/kustomize/kustomize/v3@v3.5.4 ;\
---
> 	GOFLAGS="" go get sigs.k8s.io/kustomize/kustomize/v3@v3.5.4 ;\
~~~

And then, `make deploy-setup` goes through.

--- a/Makefile
+++ b/Makefile
@@ -129,7 +129,7 @@ ifeq (, $(shell which controller-gen))
 	CONTROLLER_GEN_TMP_DIR=$$(mktemp -d) ;\
 	cd $$CONTROLLER_GEN_TMP_DIR ;\
 	go mod init tmp ;\
-	go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.3.0 ;\
+	GOFLAGS="" go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.3.0 ;\
 	rm -rf $$CONTROLLER_GEN_TMP_DIR ;\
 	}
 CONTROLLER_GEN=$(GOBIN)/controller-gen
@@ -144,7 +144,7 @@ ifeq (, $(shell which kustomize))
 	KUSTOMIZE_GEN_TMP_DIR=$$(mktemp -d) ;\
 	cd $$KUSTOMIZE_GEN_TMP_DIR ;\
 	go mod init tmp ;\
-	go get sigs.k8s.io/kustomize/kustomize/v3@v3.5.4 ;\
+	GOFLAGS="" go get sigs.k8s.io/kustomize/kustomize/v3@v3.5.4 ;\
 	rm -rf $$KUSTOMIZE_GEN_TMP_DIR ;\
 	}
 KUSTOMIZE=$(GOBIN)/kustomize

